lib/puma/request.rb

```diff
@@ -172,40 +172,42 @@ def prepare_response(status, headers, res_body, requests, client)
       content_length = resp_info[:content_length]
       keep_alive     = resp_info[:keep_alive]
 
-      # below converts app_body into body, dependent on app_body's characteristics, and
-      # resp_info[:content_length] will be set if it can be determined
-      if !content_length && !resp_info[:transfer_encoding] && status != 204
-        if res_body.respond_to?(:to_ary) && (array_body = res_body.to_ary) && array_body.is_a?(Array)
-          body = array_body
-          content_length = body.sum(&:bytesize)
-        elsif res_body.is_a?(File) && res_body.respond_to?(:size)
-          body = res_body
-          content_length = body.size
-        elsif res_body.respond_to?(:to_path) && res_body.respond_to?(:each) &&
+      if res_body.respond_to? :each
+        # below converts app_body into body, dependent on app_body's characteristics, and
+        # resp_info[:content_length] will be set if it can be determined
```

Comment:
Although the comment says we are setting the `resp_info[:content_length]` we set the variable content_length. Should we adjust the comment?

Reply:
I remember thinking that needed a change, but I guess something interrupted my thought?  Fixed in [Fix 'Did I do that?' mistakes...](https://github.com/puma/puma/pull/3072/commits/133c069f38de1c0ca699538fa6c3e39780aefdb9)

Thanks for reviewing.